contributed_definitions/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1049 @@
+                      detector assembly. But there are irregular detectors. In this
+                      case the distance must be specified for each detector pixel.
+                    </doc>
+                </field>
+                <field name="efficiency" type="NX_FLOAT" units="NX_DIMENSIONLESS" optional="true">
+                    <doc>efficiency of the detector</doc>
+                        <dimensions rank="2">
+                            <dim index="1" value="i" />
+                            <dim index="2" value="j" />
+                        </dimensions>
+                </field>
+                <field name="wavelength" type="NX_FLOAT" units="NX_WAVELENGTH" optional="true">
+                    <doc>
+                        This field can be two things:
+
+                        1. For a pixel detector it provides the nominal wavelength
+                        for which the detector has been calibrated.
+
+                        2. For other detectors this field has to be seen together with
+                        the efficiency field above.
+                        For some detectors, the efficiency is wavelength dependent.
+                        Thus this field provides the wavelength axis for the efficiency field.
+                        In this use case, the efficiency and wavelength arrays must
+                        have the same dimensionality.
+                    </doc>
+                    <dimensions rank="2">
+                      <dim index="1" value="i" />
+                      <dim index="2" value="j" />
+                    </dimensions>
+                </field>
+                <field name="dead_time" type="NX_FLOAT" units="NX_TIME" optional="true">
+                    <doc>Detector dead time</doc>
+                    <dimensions rank="3">
+                        <dim index="1" value="nP" />
+                        <dim index="2" value="i" />
+                        <dim index="3" value="j" />
+                    </dimensions>
+                </field>
+                <field name="count_time" type="NX_NUMBER" units="NX_TIME" optional="true">
+                    <doc>Elapsed actual counting time</doc>
+                    <dimensions rank="1">
+                        <dim index="1" value="nP" />
+                    </dimensions>
+                </field>
+                <field name="depends_on" type="NX_CHAR" optional="true">
+                    <doc>
+                        The axis on which the detector position depends may be stored
+                        anywhere, but is normally stored in the *NXtransformations
+                        group* within the *NXdetector group*.
+                    </doc>
+                </field>
+                <group type="NXtransformations" minOccurs="0">
+                    <doc>
+                        This is the recommended location for detector goniometer

```

Comment:
Do these experiments include goniometers?

Reply:
They might. @aapaecklar can provide details.

Reply:
Dear all, for the moment we only consider cradles for the eventual sin2psi method (1 ax rotation only)